src/components/TileGallery/TileGallery.jsx

```diff
@@ -1,12 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import TileGallerySection from './TileGallerySection';
-
 const propTypes = {
   /** Component children's to be rendered */
-  children: PropTypes.arrayOf(PropTypes.shape({ type: PropTypes.oneOf([TileGallerySection]) }))
-    .isRequired,
+  children: PropTypes.node.isRequired,
```

Comment:
Memory is fuzzy on the details here, but I think we were trying to explictly require that a TileGallerySection be used as children here instead of allowing anything. Aiming for total adherence to the design spec.

What was the warning?

Reply:
First, the `children` prop here is actually returning an object, rather than an array. After removing the `arrayOf`, there is another issue in which there is a surrounding React fragment meaning the children would never equal `TileGallerySection` unless it is a single instance of it with no parent

Reply:
Ah yeah just took a look at the story. Even further - I'm not sure why this is a component if it's only job is to wrap children in a class. Beyond the scope here though. Looks good 👍